src/components/manifold-oauth/manifold-oauth.tsx

```diff
@@ -29,10 +37,19 @@ export class ManifoldOauth {
     window.removeEventListener('message', this.tokenListener);
   }
 
+  refreshIframe() {
+    const iframe = this.el.querySelector('iframe');
+    if (iframe) {
+      const src = iframe.getAttribute('src');
+      // set iframe to its own src to refresh. Works in Chrome, Firefox, and Safari.
+      iframe.setAttribute('src', src);
+    }
+  }
```

Comment:
Based on past ~failures~ experience here, I’ve seen our endpoint err when given a search param that it didn’t expect. In light of that, I wanted to see if I could avoid that entirely. I tested this in Chrome, FF, and Safari, and resetting `src` does indeed create a new request.

Since that’s all we want—creating a new request, not dealing with search params—I figured this is the simpler, more straightforward option.

Reply:
For more info: the reason why we needed to do `?ts` was because Stencil would memoize & block prop changes if they were the same string. But since we are no longer dealing with Stencil’s memoization, this is a simpler solution.